sdk/cognitivelanguage/azure-ai-language-questionanswering/azure/ai/language/questionanswering/aio/operations/_question_answering_client_operations.py

```diff
@@ -37,24 +38,100 @@ async def query_knowledgebase(
     ) -> "_models.KnowledgeBaseAnswers":
         """Answers the specified question using your knowledge base.
 
-        Answers the specified question using your knowledge base.
-
+        :keyword project_name: The name of the project to use.
```

Comment:
nit: the param `knowledge_base_query_options` doc should come first.

This might be autorest related (if so, my bad). It was generating out of order at one point, but I've fixed it now

Reply:
No this is likely a copy+paste fail - thanks!